src/components/Input.react.js

```diff
@@ -25,25 +25,29 @@ export default class Input extends Component {
             setProps,
             type
         } = this.props;
-        const {value} = this.state;
+        //const {value} = this.state;
```

Comment:
If you comment this out, it will mean that a Dash user cannot provide the `value` prop to the `dcc.Input` component (or rather, it will not do anything). You can see in the `constructor` of the class that `this.state.value` is set to `props.value`, and below in `<input />`, it gets set to the value attribute: `<input value={value} />`.